tests/src/Functional/RdfEntityTypeCreation.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Drupal\Tests\rdf_entity\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests the creation of RdfEntityType entities (RDF bundles)
+ *
+ * @group rdf_entity
+ */
+class RdfEntityTypeCreation extends BrowserTestBase {
```

Comment:
Let's call it `RdfEntityTypeCreationTest` so that it honors the age old traditions of test naming in Drupal.